lms/templates/courseware/progress.html

```diff
@@ -50,23 +50,25 @@
         <h1>${_("Course Progress for Student '{username}' ({email})").format(username=student.username, email=student.email)}</h1>
       </header>
 
-      %if settings.FEATURES.get("SHOW_PROGRESS_SUCCESS_BUTTON"):
```

Comment:
Please remove these unused settings (I'll communicate this change to the community):

* `SHOW_PROGRESS_SUCCESS_BUTTON`
* `PROGRESS_SUCCESS_BUTTON_URL`
* `PROGRESS_SUCCESS_BUTTON_TEXT_OVERRIDE`